packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -128,65 +128,5 @@ describe.skipIf(!process.env.RUNNING_IN_CI)(
       // Verify decryption
       expect(decryptedMessageString).toEqual(messageString);
     }, 15000); // 15s timeout
-
-    test('should encrypt and decrypt according to wallet allowlist condition', async () => {
```

Comment:
This test is no more working. And it has been already deleted at https://github.com/nucypher/taco-web/pull/688.